assets/js/components/SapSystemDetails/GenericSystemDetails.test.jsx

```diff
@@ -92,4 +95,59 @@ describe('GenericSystemDetails', () => {
       screen.getAllByTestId('eos-svg-component');
     expect(health).toHaveClass('fill-black');
   });
+
+  it.each([
+    {
+      type: APPLICATION_TYPE,
+      text: 'In the case of an ASCS instance',
+      field: 'application_instances',
+      row: 0,
+    },
+    {
+      type: DATABASE_TYPE,
+      text: 'In the case of the last database instance',
+      field: 'database_instances',
+      row: 1,
+    },
+  ])(
+    'should clean up $type instance on request',
+    async ({ type, text, field, row }) => {
+      const user = userEvent.setup();
+      const mockedCleanUp = jest.fn();
+
+      const sapSystem = sapSystemFactory.build();
+
+      sapSystem.database_instances[0].absent_at = faker.date
```

Comment:
This looks a bit cumbersone. This view shouldn't never have db and app instances all together